noether_examples/CMakeLists.txt

```diff
@@ -3,57 +3,75 @@ cmake_minimum_required(VERSION 2.8.3)
 ## Compile as C++14,
 add_compile_options(-std=c++14)
 
-
 project(noether_examples)
 
-
 find_package(VTK 7.1 REQUIRED NO_MODULE)
 include(${VTK_USE_FILE})
 
-find_package(catkin REQUIRED cmake_modules
-    mesh_segmenter
-    tool_path_planner
-    path_sequence_planner
-    vtk_viewer
-    noether_msgs
-    roscpp
-    pcl_conversions
-    roslib
-    actionlib
-    noether_conversions
+find_package(catkin REQUIRED COMPONENTS
+  actionlib
+  cmake_modules
+  mesh_segmenter
+  noether_conversions
+  noether_msgs
+  path_sequence_planner
+  pcl_conversions
+  roscpp
+  roslib
+  tool_path_planner
+  vtk_viewer
 )
 
 catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES noether_examples
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    ${PROJECT_NAME}
   CATKIN_DEPENDS
+    actionlib
     mesh_segmenter
-    tool_path_planner
-    path_sequence_planner
-    vtk_viewer
     noether_msgs
-    actionlib
+    path_sequence_planner
+    pcl_conversions
     roslib
     roscpp
-    pcl_conversions
-  DEPENDS VTK
+    tool_path_planner
+    vtk_viewer
+  DEPENDS
+    VTK
 )
 
 include_directories(
-    include
-    ${catkin_INCLUDE_DIRS})
-
-
-add_executable(mesh_segmenter_client_node src/mesh_segmenter_client.cpp)
+  include
+  ${catkin_INCLUDE_DIRS}
+)
 
+add_executable(mesh_segmenter_client_node
+  src/mesh_segmenter_client.cpp
+)
```

Comment:
Do these not need the exported_targets depends?

Reply:
It builds fine without it.  What made the difference here was making sure to call install() in the included package.